seatunnel-core/seatunnel-core-base/src/main/java/org/apache/seatunnel/config/command/CommandSparkArgs.java

```diff
@@ -57,4 +59,11 @@ public boolean isTestConfig() {
         return testConfig;
     }
 
+    public String getMaster() {
```

Comment:
Maybe we can assign a default value for the `master` variable to improve the ease of use, such as `local`. What do you think

Reply:
make sense, but we can achive this goal in another issue.